packages/terra-demographics-banner/tests/nightwatch/demographics-banner-spec.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-unused-expressions */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  afterEach: (browser, done) => {
```

Comment:
The error being thrown is actually the default nightwatch test added by an older version of the generator.

terra-base also has this issue: https://github.com/cerner/terra-core/blob/master/packages/terra-base/tests/nightwatch/base-spec.js